examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,172 @@
+{
+"metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": [{
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
+            "ru": "Эбботт, Эдвин Эбботт",
+            "en": "Abbott, Edwin Abbott"
+        },
+    }, {
+        "name": "Bob Braggert",
+        "sort_as": "Braggert, Bob"
+    }],
+    "narrator": [
+        {
+        "name": "Abe Narrator",
+        "scheme": "https://www.loc.gov/marc/relators/",
```

Comment:
No need for `scheme` here or in the other elements since we're not using contributor.